mediator/fd_nems.yaml

```diff
@@ -1,808 +0,0 @@
- field_dictionary:
```

Comment:
I think that this file is cleaned. We did couple of changes in this file for WW3 coupling and probably those fields (adding new aliases for 10 wind components and roughness length) needs to be included to field dictionary again.